### DISCUSSION TITLE
Structure of `PrismaClientKnownRequestError`'s `meta` field

### DISCUSSION THREAD

**Discussion by zhulien-ivanov** [prisma/prisma#20815] [Q&A]
### Question

Where can we find detailed information of the structure of `meta`'s field? I can't find any related information neither in the docs, nor here in discussions and I notice that each error returns a different `meta` structure. I've also tried finding something in the source code but I have to dig into the query engines which is unreasonable.

For example: 
`P2002` returns `meta.target` of type `string[]`.
`P2003` returns `meta.field_name` of type `string`.

Currently, I have to try to simulate every error in order to see what the structure of `meta` might be. 

This should be specified in the related docs here: https://www.prisma.io/docs/reference/api-reference/error-reference.

**Comment by zhulien-ivanov** [comment]
Really, there is no official documentation on this?

**Reply by ludralph** [reply]
Hi @zhulien-ivanov 👋 

Thank you for raising this question. At the moment this is not documented. However, you can create a docs [issue](https://github.com/prisma/docs/issues/new?assignees=&labels=docs&projects=&template=doc_request.md) so that we can take steps to document this.

**Reply by zhulien-ivanov** [reply]
Will do. Thank you!
